uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java

```diff
@@ -165,6 +174,7 @@ public static void copyFileToFolderInSdCard(final File file, @Nullable final Str
      * @param action action is which operation to be performed.
      */
     public static void launchIntent(@NonNull final String action) {
+        Logger.i(TAG, "Launches an activity specified by the action..");
```

Comment:
I don't like statements here that sound more like a javadoc type of description such as `"Launches an activity specified by the action.."`

I would rather prefer logs to be more like telling what is happening at the present time such as telling a sequence of events as they happen - something like: `"Launching an activity specified by the action: " + action`

I would give the same feedback for all other places

Reply:
A question here for @ayushmeghwani and @shahzaibj do we want all of this logging at info level, also appending package names and other metadata to the log statements, would it be better to have them at verbose level, just a thought?

Reply:
According to my understanding @tanmaymanolkar1 the main purpose of logging in automation infrastructure and test cases is that when we see the automation files generated at FireBase Test Labs, we can see the state of Test Case, and these automation log files will kind of help us to see that where things went wrong.
And then these kind of logging should be done at verbose level or info level ? I will say I  have to look more on that and read some resources about it and will get back to you on that.

Reply:
As discussed the other day, we can move these to `info` level

Reply:
yes.